docs/current_docs/integrations/snippets/openshift.yaml

```diff
@@ -4,15 +4,15 @@ fullnameOverride: ""
 engine:
   image:
     repository: registry.dagger.io/engine
-    tag: v0.10.2
+    tag: v0.11.0
```

Comment:
Can we change this to `latest`? It would simplify maintenance of this page but would it work? @chrira @gerhard ?

Reply:
OK.